spec/pactas_itero/client_spec.rb

```diff
@@ -194,7 +194,7 @@
     it 'uses the sandbox endpoint by default' do
       client = PactasItero::Client.new
 
-      expect(client.api_endpoint).to eq 'https://sandbox.pactas.com/'
+      expect(client.api_endpoint).to eq 'https://sandbox.billwerk.com/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.